validator/validator-main.protoascii

```diff
@@ -3145,6 +3145,25 @@ tags: {
     }
   }
 }
+# AMP IMA VIDEO
+tags: {
+  html_format: AMP
+  tag_name: "SCRIPT"
+  spec_name: "amp-ima-video > script[type=application/json]"
+  unique: true
+  mandatory_parent: "AMP-IMA-VIDEO"
+  attrs: {
```

Comment:
Can the dispatch key be on `type`? seems to work for `amp-analytics`: https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/validator-amp-analytics.protoascii#L35